rust/Cargo.toml

```diff
@@ -46,6 +46,8 @@ url = "2.3"
 rand = "0.8.3"
 futures = "0.3"
 uuid = { version = "1.2", features = ["v4"] }
+faiss = {version = "0.11.0", features=["gpu"], optional = true }
```

Comment:
Does this make the binary much larger?

Reply:
this needs to be able to locate `faiss_c.so` somewhere, so it does not make the binary larger, but crash if `faiss.so` not found.